CMakeLists.txt

```diff
@@ -387,12 +387,9 @@ set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTDIR}/espressomd")
 #######################################################################
 
 # C++ standard
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
I'm all in favour of bumping the standard version to 14, but I think we should so also
for the CUDA. So you would have to check which CUDA version is needed for 14,
and change that accordingly. Also the documentation has to be adapted in several
places and some code can be removed form utils, because it is now provided by the
language. If this is not strictly needed here, I think it should be a separate PR.

Reply:
CUDA 8 supports C++14. That means dropping support for Ubuntu 16.04's CUDA packages (but one can manually install newer ones).

Reply:
i think the only reason for this was ```constexpr std::pair``` in utils/u32_to_u64.hpp

Reply:
Ok, that's not enough to justify the switch. You could just do `reinterpret_cast`s between `uint32_t[2]` and `uint64_t` instead of using `std::pair`.